final_task/rss_reader/save_in_format_functions.py

```diff
@@ -47,7 +47,7 @@ def save_in_html(path, news_list):
     for new in news_list:
         html_content += get_new_content_html(new)
     html_content += "</body>\n</html>"
-    with open(path, 'w', encoding="utf-8") as html_file:
+    with open(os.path.join(path, "news.html"), 'w', encoding="utf-8") as html_file:
```

Comment:
It is better to have more informative name for news file. For example date-time-source and etc.
Same for fb2 format

Reply:
Can't find this reduntant symbols. Check your encoding, it must be utf8. 
You mean, for example, that file should be saved with date of run application? Today file should be like 20191121-time_of_run-source.fb2?

Reply:
Please try to parse the whole feed to the html and use Firefox browser to check it. (Linux OS)

Reply:
I got all news from yahoo.com to fb2, html and open them in notepad++, opera and firefox and don't met this symbols anywhere